src/cli_plugin/install/native_controllers_prompt.js

```diff
@@ -0,0 +1,30 @@
+import readline from 'readline';
+import { get } from 'lodash';
+
+export async function nativeControllersPrompt(settings) {
+  if (settings.plugins.some(plugin => get(plugin, 'kibana.nativeControllers'))) {
+    const rl = readline.createInterface({
+      input: process.stdin,
+      output: process.stdout
+    });
+
+
+    const question = `
+@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+@         WARNING: plugin forks a native controller          @
+@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+This plugin launches a native controller that is not subject
+to the system call filters. This plugin is capable of spawning
+additional processes and running arbitrary system commands.
```

Comment:
Other plugins are still capable of running arbitrary code. (The plugin can write a new plugin with a controller and then kill Kibana to get it restarted with that process)

The seccomp is a great start, but it's not a hard-to-escape sandbox. This _could_ by omission suggest that other plugins can't – which (as far as we know) is the case with ES, which I think this message is inspired by. :)

(The seccomp filtering is definitely a super useful start, and I know this isn't trying to be a sandbox at this point)

Reply:
@alexbrasetvik do you have any suggestion for the wording of this warning that would make this situation more clear?

Reply:
@kobelb Maybe show a warning for _all_ plugins? "Plugins are a security risk, you should only install plugins from sources you trust"?

Seccomp helps against accidental compromise, but an intentionally malicious plugin will get around it, so plugins with native controllers aren't really that more dangerous than plugins without them.

Reply:
I think adding the warning when we install any plugin makes sense, but we should also be warning the user when the install a plugin that have a native controller as we're explicitly allowing it to exist outside of the limited system call filtering that we put in place, with the intent of it being able to spawn child processes, even after more effort is put into sandboxing the plugins.

Reply:
I revised the warning so users won't infer that we're guaranteeing that other plugins aren't capable of spawning additional processes or performing other malicious behavior.